### PR TITLE
LPS-94506 blogs-test: Create BlogsEntryMultiLanguageSearchTest

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

File: modules/apps/blogs/blogs-test/src/testIntegration/java/com/liferay/blogs/search/test/BlogsEntryMultiLanguageSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import com.liferay.portal.kernel.search.Document;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.rule.SynchronousDestinationTestRule;
+import com.liferay.portal.kernel.util.LocaleThreadLocal;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.search.test.util.FieldValuesAssert;
+import com.liferay.portal.search.test.util.IndexerFixture;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerTestRule;
```
Comment (foshiro-zz) [inline modules/apps/blogs/blogs-test/src/testIntegration/java/com/liferay/blogs/search/test/BlogsEntryMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

File: modules/apps/blogs/blogs-test/src/testIntegration/java/com/liferay/blogs/search/test/BlogsEntryMultiLanguageSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+ */
+@RunWith(Arquillian.class)
+public class BlogsEntryMultiLanguageSearchTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerTestRule.INSTANCE,
```
Comment (foshiro-zz) [inline modules/apps/blogs/blogs-test/src/testIntegration/java/com/liferay/blogs/search/test/BlogsEntryMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

**Review by foshiro-zz — CHANGES_REQUESTED** [review]
@vagner-castro Code and tests looks good to me. Just have to change PermissionCheckerTestRule to PermissionCheckerMethodTestRule

**Comment by foshiro** [comment]
ci:test:sf

**Comment by vagnerbc** [comment]
Changes updated. I have to do more things becouse the tests was broking.

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

**Review by foshiro-zz — APPROVED** [review]

**Comment by foshiro-zz** [comment]
Submitted to https://github.com/brandizzi/liferay-portal/pull/777

**Pull request closed without merging**